docs/docs/path-prefix.md

```diff
@@ -36,3 +36,11 @@ module.exports = {
   pathPrefix: `/blog`
 }
 ```
+
+Then pass `--prefix-paths` cmd option to Gatsby.
+
+```sh
+gatsby build --prefix-paths
+```
+
+NOTE: When using only `build`, Gatsby ignores `pathPrefix`.
```

Comment:
Perhaps re-word this to:

"NOTE: When running the command without the `--prefix-paths` flag, Gatsby ignores your `pathPrefix`."